Base/Python/slicer/util.py

```diff
@@ -156,7 +156,7 @@ def showStatusMessage(message, duration = 0):
   mw.statusBar().showMessage(message, duration)
   return True
 
-def findChildren(widget=None, name="", text="", title="", className=""):
```

Comment:
Out of curiosity, what is the use case ? Is it just for convenience when using the python interactor ? 

I would advise against using the tooltip for production code ... it would be too brittle (e.g could break unexpectedly after fixing a typo in a tooltip).